plugin/pkg/auth/authorizer/node/graph_populator.go

```diff
@@ -155,8 +156,13 @@ func (g *graphPopulator) updatePod(oldObj, obj interface{}) {
 			return
 		}
 	}
-	klog.V(4).Infof("updatePod %s/%s for node %s", pod.Namespace, pod.Name, pod.Spec.NodeName)
+
+	startTime := time.Now()
+	klog.V(5).Infof("updatePod %s/%s for node %s started at %s", pod.Namespace, pod.Name, pod.Spec.NodeName, startTime)
 	g.graph.AddPod(pod)
+
+	endTime := time.Now()
+	klog.V(5).Infof("pod %s in namespace %s is updated for the node %s completed at %s", pod.Name, pod.Namespace, pod.Spec.NodeName, endTime)
```

Comment:
Sorry if my suggestion wasn't clear, I meant that explicitly recording the duration seems preferable:

```suggestion
	klog.V(4).Infof("updatePod %s/%s for node %s", pod.Namespace, pod.Name, pod.Spec.NodeName)
	startTime := time.Now()
	g.graph.AddPod(pod)
	klog.V(5).Infof("updatePod %s/%s for node %s completed in %v", pod.Name, pod.Namespace, pod.Spec.NodeName, time.Since(startTime))
```

Same below.

Reply:
you wanted to like this, maybe I misunderstood that.
Thanks, @tallclair for reviewing it.  I will update it according to suggestions!

Reply:
Hi @tallclair
Done the changes, PTAL!